tools/bug-report/pkg/cluster/cluster.go

```diff
@@ -45,115 +45,109 @@ func ParsePath(path string) (namespace string, deployment, pod string, container
 
 // shouldSkip means that current pod should be skip or not based on given --include and --exclude
 func shouldSkip(deployment string, config *config2.BugReportConfig, pod *corev1.Pod) bool {
-	var isInclude bool = len(config.Include) > 0
-	var isExclude bool = len(config.Exclude) > 0
-
-	if isExclude {
```

Comment:
I just removed `isInclude` and `isExclude` from the code. GitHub shows the diff to be large due to the removal of this if condition.